test/src/main/java/org/apache/accumulo/test/functional/CompactionIT.java

```diff
@@ -494,6 +495,54 @@ public void testSuccessfulCompaction() throws Exception {
     }
   }
 
+  @Test
+  public void testMultiStepCompactionThatDeletesAll() throws Exception {
+
+    // There was a bug where user compactions would never complete when : the tablet had to be
+    // compacted in multiple passes AND the intermediate passes produced no output.
+
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+      final String tableName = getUniqueNames(1)[0];
+      c.tableOperations().create(tableName);
+      c.tableOperations().setProperty(tableName, Property.TABLE_MAJC_RATIO.getKey(), "100.0");
+
+      var beforeCount = countFiles(c);
+
+      try (var writer = c.createBatchWriter(tableName)) {
+        for (int i = 0; i < 60; i++) {
```

Comment:
A variable with a descriptive name could be added in place of `60` here to make it easier to tell what it represents when it is referenced throughout this test.